src/social/__tests__/__snapshots__/SocialIcon.js.snap

```diff
@@ -3,8 +3,6 @@
 exports[`SocialIcon component should render light social icon 1`] = `
 <View
   disabled={false}
-  onLongPress={[Function]}
```

Comment:
Removed `TouchableHighlight` cases from snapshot.

Reply:
Unneeded props removed to avoid warnings.